Api/RequestInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright Elgentos BV. All rights reserved.
+ * https://www.elgentos.nl/
+ */
+
```

Comment:
Dont think the Elgentos copyright should be here :)

Reply:
Whoops my bad :)